moped-editor/src/views/projects/newProjectView/NewProjectView.js

```diff
@@ -177,33 +175,8 @@ const NewProjectView = () => {
         // On success
         .then(response => {
           // Destructure the data we need from the response
```

Comment:
this comment can be removed now!

Was this deleted code legacy from when you could add features in the new project flow? If so, my bad for not removing it before when I simplified the new project view

Reply:
ooh good catch. no, this was code was in place after the simplified project view—it was part of a callback mutation that populated the many-to-many moped_proj_features_components table.

Reply:
🕵️‍♀️